eagleproject/core/blockchain/harvest/transaction_history.py

```diff
@@ -252,11 +285,14 @@ def calculate_difference(current_stat, previous_stat):
     supply_data = json_report["supply_data"] if "supply_data" in json_report else None
     supply_data_previous = json_report_previous["supply_data"] if "supply_data" in json_report_previous else None
 
+    ogv_data = json_report["ogv_data"] if "ogv_data" in json_report else None
```

Comment:
nit: Assuming that `json_report` is a dict, why not use `json_report.get("ogv_data")`? The `.get()` method returns the value of the key if it exists or `None` otherwise

Reply:
I think json_report is a list